Examples/Applications/GeometryViewer/index.js

```diff
@@ -202,6 +207,14 @@ function createPipeline(fileName, fileContents) {
   controlContainer.appendChild(colorBySelector);
   controlContainer.appendChild(componentSelector);
   controlContainer.appendChild(opacitySelector);
+
+  if (
+    navigator.xr !== undefined &&
+    navigator.xr.isSessionSupported('immersive-ar') &&
+    fullScreenRenderWindow.getApiSpecificRenderWindow().getXrSupported()
```

Comment:
`Applications/GeometryViewer` will only include a button for AR if 1) WebXR is available, 2) the device supports AR, and 3) the rendering backend supports AR (aka WebGL and not WebGPU)